crates/iceberg/src/scan.rs

```diff
@@ -186,6 +239,27 @@ impl TableScan {
         .boxed())
     }
 
+    fn create_manifest_eval_factory(
```

Comment:
```suggestion
    fn create_manifest_evaluator(
```
I think it's creating `ManifestEvaluator` rather than factory?

Reply:
Aah yes, the variable name has been left over from the previous name for the `ManifestEvaluator`. Will fix